python/cuspatial/cuspatial/core/geoseries.py

```diff
@@ -536,3 +563,57 @@ def to_arrow(self):
                 arrow_polygons,
             ],
         )
+
+    def contains(self, other, align=True):
```

Comment:
Refactor contains from `geoseries.py` into `core/binops/contains.py`. This will contribute to designing the factoring of the abstraction for binops. I don't want `GeoSeries.py` to be cluttered with binops code, it should live in its own directory.